src/app/components/datepicker/index.html

```diff
@@ -30,22 +31,23 @@ <h3 class="sky-text-success">Use when</h3>
 	<sky-demo-page-content sectionHeading="Options">
 		<h3>Fuzzy dates</h3>
       <div class="sky-text-block">
-        <p>Fuzzy dates let the user enter a partial date. Supported formats include:</p>
-		    <ul>
+        <p>Fuzzy dates allow users to specify dates that are not complete. They enable partial dates when users are unsure of the complete date. For example, if users know the year but not the month or day, they can enter just the year. Supported formats include:</p>
```

Comment:
I tried to flesh this out a little to address https://github.com/blackbaud/skyux2-docs/issues/683. I'm not sure how much more we really need to say about fuzzy dates. I think this is enough, based on https://kb.blackbaud.com/articles/Article/41804.